### PR TITLE
[Bug] Fix description of rare candy to have proper amount of level

### PR DESCRIPTION
**Pull request by hayuna**
## What are the changes?
Update description for rare candy / rare candy

## Why am I doing these changes?
Before rare candy always display 1 LV

## What did change?
Add variable `levels` to description. `levels` = amount of candy jars + 1

### Screenshots/Videos
Screenshot 2024-07-07 at 22 19 32
Screenshot 2024-07-07 at 22 33 21

## How to test the changes?
- Get Candy Jar
- Get Rare Candy / Rarer candy
- see description of item in store
To speed up you can boost rate of these items to get from store

```javascript
    new WeightedModifierType(modifierTypes.CANDY_JAR, 1000),
    new WeightedModifierType(modifierTypes.RARE_CANDY, 1000),
    new WeightedModifierType(modifierTypes.RARER_CANDY, 1000),
```

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by hayuna** [comment]
Seems like updating locales by replacing 1 into variable will be easy, but some countries are using not a number, but "one"

**Comment by flx-sta** [comment]
@hayuna please provide a setup for every language.
Worst things worst just replace the current translation with the English one. We can then call in the localization squad to get the proper translations set up in minimum time

File: src/locales/de/modifier-type.ts [hunk 1]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "Erhöht die Stärke aller {{moveType}}-Attacken eines Pokémon um 20%.",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "Erhöht das Level eines Pokémon um 1.",
+      description: "Increases a Pokémon's level by {{levels}}.",
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.ts:61]:
```suggestion
      description: "Erhöht das Level eines Pokémon um {{levels}}.",
```

File: src/locales/de/modifier-type.ts [hunk 2]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
     },
     "AllPokemonLevelIncrementModifierType": {
-      description: "Erhöht das Level aller Teammitglieder um 1.",
+      description: "Increases all party members' level by {{levels}}.",
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.ts:64]:
```suggestion
      description: "Erhöht das Level aller Teammitglieder um {{levels}}.",
```

File: src/locales/fr/modifier-type.ts [hunk 3]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "Augmente de 20% la puissance des capacités de type {{moveType}} d’un Pokémon.",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "Fait monter un Pokémon d’un niveau.",
+      description: "Increases a Pokémon's level by {{levels}}.",
     },
     "AllPokemonLevelIncrementModifierType": {
-      description: "Fait monter toute l’équipe d’un niveau.",
+      description: "Increases all party members' level by {{levels}}.",
```
Comment (Adri1) [inline src/locales/fr/modifier-type.ts:64]:
```suggestion
      description: "Fait monter toute l’équipe de {{levels}} niveau·x.",
```

File: src/locales/fr/modifier-type.ts [hunk 4]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "Augmente de 20% la puissance des capacités de type {{moveType}} d’un Pokémon.",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "Fait monter un Pokémon d’un niveau.",
+      description: "Increases a Pokémon's level by {{levels}}.",
```
Comment (Adri1) [inline src/locales/fr/modifier-type.ts:61]:
```suggestion
      description: "Fait monter un Pokémon de {{levels}} niveau·x.",
```

File: src/locales/fr/modifier-type.ts (hunk 3 above)
Comment (Adri1) [inline src/locales/fr/modifier-type.ts:64]:
```suggestion
      description: "Fait monter toute l’équipe de {{levels}} niveau·x.",
```

**Review by Adri1 — COMMENTED** [review]
Added French

**Review by Adri1 — APPROVED** [review]

File: src/locales/ko/modifier-type.ts [hunk 5]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "지니게 하면 {{moveType}}타입 기술의 위력이 20% 상승한다.",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "포켓몬 1마리의 레벨이 1만큼 상승한다.",
+      description: "Increases a Pokémon's level by {{levels}}.",
```
Comment (EnochG1) [inline src/locales/ko/modifier-type.ts:61]:
```suggestion
      description: "포켓몬 1마리의 레벨이 기본 1만큼, 사탕단지의 개수에 따라 최대 {{levels}}까지 상승한다.",
```

File: src/locales/ko/modifier-type.ts [hunk 6]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
     },
     "AllPokemonLevelIncrementModifierType": {
-      description: "자신의 모든 포켓몬의 레벨이 1씩 상승한다.",
+      description: "Increases all party members' level by {{levels}}.",
```
Comment (EnochG1) [inline src/locales/ko/modifier-type.ts:64]:
```suggestion
      description: "자신의 모든 포켓몬의 레벨이 기본 1씩, 사탕단지의 개수에 따라 최대 {{levels}}까지 상승한다.",
```

**Review by EnochG1 — COMMENTED** [review]
Korean translation

File: src/locales/zh_CN/modifier-type.ts [hunk 7]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "一只宝可梦的{{moveType}}系招式威力提升20%。",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "一只宝可梦等级提升1级。",
+      description: "Increases a Pokémon's level by {{levels}}.",
```
Comment (40chyan) [inline src/locales/zh_CN/modifier-type.ts:61]:
```suggestion
      description: "使一只宝可梦的等级提升{{levels}}级。",
```

File: src/locales/zh_CN/modifier-type.ts [hunk 8]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
     },
     "AllPokemonLevelIncrementModifierType": {
-      description: "所有成员宝可梦等级提升1级。",
+      description: "Increases all party members' level by {{levels}}.",
```
Comment (40chyan) [inline src/locales/zh_CN/modifier-type.ts:64]:
```suggestion
      description: "使一只寶可夢的等級提升{{levels}}級。",
```

File: src/locales/pt_BR/modifier-type.ts [hunk 9]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "Aumenta o poder dos ataques do tipo {{moveType}} de um Pokémon em 20%.",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "Aumenta em 1 o nível de um Pokémon.",
+      description: "Increases a Pokémon's level by {{levels}}.",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/modifier-type.ts:61]:
```suggestion
      description: "Aumenta em {{levels}} o nível de um Pokémon.",
```

File: src/locales/pt_BR/modifier-type.ts [hunk 10]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
     },
     "AllPokemonLevelIncrementModifierType": {
-      description: "Aumenta em 1 os níveis de todos os Pokémon.",
+      description: "Increases all party members' level by {{levels}}.",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/modifier-type.ts:64]:
```suggestion
      description: "Aumenta em {{levels}} o nível de todos os membros da equipe.",
```

File: src/locales/es/modifier-type.ts [hunk 11]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "Aumenta la potencia de los movimientos de tipo {{moveType}} de un Pokémon en un 20%.",
     },
     "PokemonLevelIncrementModifierType": {
-      description: "Aumenta el nivel de un Pokémon en 1.",
+      description: "Increases a Pokémon's level by {{levels}}.",
```
Comment (Arxxer) [inline src/locales/es/modifier-type.ts:61]:
```suggestion
      description: "Aumenta el nivel de un Pokémon en {{levels}}.",
```

File: src/locales/es/modifier-type.ts [hunk 12]
```diff
@@ -58,10 +58,10 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
     },
     "AllPokemonLevelIncrementModifierType": {
-      description: "Aumenta el nivel de todos los miembros del equipo en 1.",
+      description: "Increases all party members' level by {{levels}}.",
```
Comment (Arxxer) [inline src/locales/es/modifier-type.ts:64]:
```suggestion
      description: "Aumenta el nivel de todos los miembros del equipo en {{levels}}.",
```

**Review by Arxxer — COMMENTED** [review]
added es-es localization

**Comment by hayuna** [comment]
Only zh-TW locale left

**Review by CodeTappert — APPROVED** [review]

**Pull request merged**